### ISSUE TITLE
Font weight getting "stuck" when first set

### ISSUE THREAD

**Issue by xtianus** [wkhtmltopdf/wkhtmltopdf#3151]
When using fonts downloaded from [fonts.google.com](https://fonts.google.com), I can't set the font-weight more than once in the page because it never changes a second time.

For example, the following HTML always has bold text:

			<p style="font-family:Rubik; font-weight:500;">This is some bold text</p>
			<p style="font-family:Rubik; font-weight:300;">This is some normal text</p>

**This is some bold text
This is some normal text**

Inverting the lines, I only get normal text:

			<p style="font-family:Rubik; font-weight:300;">This is some normal text</p>
			<p style="font-family:Rubik; font-weight:500;">This is some bold text</p>

This is some normal text
This is some bold text

The full html is this:

```
<html>
	<head>
		<link href="https://fonts.googleapis.com/css?family=Rubik:300,500" rel="stylesheet">
	</head>	
	<body style="font-size: 2em;">
		<div>
			<p style="font-family:Rubik; font-weight:300;">This is some normal text</p>
			<p style="font-family:Rubik; font-weight:500;">This is some bold text</p>
		</div>
	</body>
</html>
```

Version: wkhtmltopdf 0.12.2.2 (with patched qt)
OS: Windows 8.1
Command: wkhtmltopdf.exe test1.html test1.pdf

The attached zip contains the test files.
[test.zip](https://github.com/wkhtmltopdf/wkhtmltopdf/files/511377/test.zip)

**Comment by xtianus** [comment]
Actually this doesn't seem to be related to Google Fonts only.

**Comment by xtianus** [comment]
This has been fixed in wkhtmltopdf 0.13.0-alpha-7b36694

**Comment by akaspick** [comment]
@xtianus Did you move to using alpha or did you find a workaround solution in 0.12?  I'm experiencing this same issue, but am not able to move to the alpha release.

**Comment by akaspick** [comment]
Thought I fixed it, but it appears to be somewhat random in when the issue appears.  Gonna say I don't have this resolved in 0.12
